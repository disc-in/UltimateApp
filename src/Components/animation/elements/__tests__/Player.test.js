import React from 'react';
import { render } from '@testing-library/react-native';

import Player from '../Player';

describe('<Player />', () => {
  it('renders offense correctly', async () => {
    const { toJSON } = render(<Player baseWidth={10} number={1} type="offense" />);
    expect(toJSON()).toMatchSnapshot();
  });

  it('renders defense correctly', async () => {
    const { toJSON } = render(<Player baseWidth={10} number={1} type="defense" />);
    expect(toJSON()).toMatchSnapshot();
  });
});
