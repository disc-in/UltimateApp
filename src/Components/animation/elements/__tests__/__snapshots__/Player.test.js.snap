// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Player /> renders defense correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "textAlign": "center",
        "textAlignVertical": "center",
      },
      Object {
        "backgroundColor": "#000000",
      },
      Object {
        "borderRadius": 10,
        "height": 10,
        "width": 10,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontWeight": "bold",
      }
    }
  >
    1
  </Text>
</View>
`;

exports[`<Player /> renders offense correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "textAlign": "center",
        "textAlignVertical": "center",
      },
      Object {
        "backgroundColor": "#2a9e91",
      },
      Object {
        "borderRadius": 10,
        "height": 10,
        "width": 10,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontWeight": "bold",
      }
    }
  >
    1
  </Text>
</View>
`;
